index.html

```diff
@@ -1319,8 +1319,8 @@
           <div class="note">
             If the remote playback device is abruptly disconnected during
             playback (for example, by power loss or a network disconnection),
-            the user agent SHOULD run the steps to <a>monitor the list of
-            available remote playback devices</a> before the steps
+            the user agent is expected to run the steps to <a>monitor the list
+            of available remote playback devices</a> before the steps
```

Comment:
This note describes Web-visible behavior through the API; it should be normative IMO.  Or, it could be split into a normative piece (about always running callbacks no matter what happens) and a non-normative piece (about doing so in response to abrupt disconnection).

Reply:
OK, I had applied the same logic as above but, although it is indeed difficult to reproduce an "abrupt disconnection", it is indeed easy to test that behavior if such a disconnection occurs. I'll make the note normative instead.